src/Native/Runtime/windows/CoffNativeCodeManager.cpp

```diff
@@ -29,8 +29,6 @@
 #define UBF_FUNC_REVERSE_PINVOKE        0x08
 #define UBF_FUNC_HAS_ASSOCIATED_DATA    0x10
 
-#if defined(_TARGET_AMD64_)
```

Comment:
`_UNWIND_CODE` is x64 specific. This should stay.

Reply:
Moved `_UNWIND_CODE` under `else` (`_TARGET_X64_`) branch.